libusb.go

```diff
@@ -301,13 +301,28 @@ func (libusbImpl) getDeviceDesc(d *libusbDevice) (*DeviceDesc, error) {
 			}
 			descs := make([]InterfaceSetting, 0, len(alts))
 			for _, alt := range alts {
+
+				var extraDescriptors []*RawDescriptor
+				extra := C.GoBytes(unsafe.Pointer(alt.extra), alt.extra_length)
+				for i := 0; i != len(extra); i += int(extra[i]) {
+					block := extra[i : i+int(extra[i])]
+					bType := DescriptorType(block[1])
+
+					extraDescriptors = append(extraDescriptors,
```

Comment:
join the line for append (i.e. the &RawDescriptor literal starts in the same line)